import "sofa/pbrpc/rpc_option.proto";
package mdt.LogAgentService;

message EchoRequest {
    required string message = 1;
}
message EchoResponse {
    required string message = 1;
}

// rpc service
service LogAgentService {
    rpc Echo(EchoRequest) returns(EchoResponse);
}
option py_generic_services = true;
option cc_generic_services = true;

